T/TestU01/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "TestU01"
+version = v"1.2.3-1"
+
+# Collection of sources required to build TestU01
+sources = [
+    ArchiveSource(
+        "http://simul.iro.umontreal.ca/testu01/TestU01.zip",
+        "bc1d1dd2aea7ed3b3d28eaad2c8ee55913f11ce67aec8fe4f643c1c0d2ed1cac"
```

Comment:
This doesn't look like a good source: it isn't versioned, if the author will publish a new version the sha256 hash will become invalid.  Is this the only source we have?